fmriprep/data/registration_settings.json

```diff
@@ -0,0 +1,22 @@
+{
+    "metric": ['Mattes'] * 3 + [['Mattes', 'CC']],
```

Comment:
Oh yeah I need to work on this, im not pulling from the json file yet. Not sure how to convert some of these things to json, if they should just be strings in the value part or what.